<?xml version="1.0"?>
<launch>
    <node pkg="mcr_twist_controller" type="twist_controller_node"
            name="twist_controller" output="screen">

        <!-- the cycle time of the node (in seconds) -->
        <param name="cycle_time" type="double" value="0.1"/>

        <remap from="~controlled_velocity" to="/arm_1/arm_controller/cartesian_velocity_command"/>

    </node>
</launch>